$NetBSD: patch-ab,v 1.8 2007/09/25 01:22:15 dmcmahill Exp $

Adds missing -lsocket needed on solaris for socket(), bind(), and friends.

--- configure.in.orig	2007-08-16 12:13:34.000000000 -0400
+++ configure.in	2007-09-24 08:44:58.668063000 -0400
@@ -675,4 +675,5 @@
 if test "`uname`" = "SunOS" -o "`uname -sr`" = "HP-UX B.11.00"; then
   dnl ...for Solaris
+  AC_CHECK_LIB([socket], [socket])
   AC_CHECK_LIB([nsl], [xdr_int])
 fi
